lib/analyzer/apk-analyzer.ts

```diff
@@ -0,0 +1,74 @@
+import subProcess from '../sub-process';
+
+export {
+  analyze,
+};
+
+function analyze(targetImage) {
+  return getPackages(targetImage)
+    .then(pkgs => ({
+      Image: targetImage,
+      AnalyzeType: 'Apk',
+      Analysis: pkgs,
+    }));
+}
+
+function getPackages(targetImage) {
+  return subProcess.execute('docker', [
+    'run', '--rm', targetImage, 'cat', '/lib/apk/db/installed',
+  ])
+    .catch(stderr => {
+      if (typeof stderr === 'string' && stderr.indexOf('No such file') >= 0) {
+        return '';
+      }
+      throw new Error(stderr);
+    })
+    .then(parseFile);
+}
+
+function parseFile(text) {
+  const pkgs = [];
+  let curPkg = null;
+  for (const line of text.split('\n')) {
+    curPkg = parseLine(line, curPkg, pkgs);
+  }
+  return pkgs;
+}
+
+function parseLine(text, curPkg, pkgs) {
+  const key = text.charAt(0);
+  const value = text.substr(2);
+  switch (key) {
+    case 'P': // Package
+      curPkg = {
+        Name: value,
+        Version: null,
+        Source: null,
+        Provides: [],
+        Deps: {},
+        AutoInstalled: null,
```

Comment:
there is no place we change this value - didn't we implement this logic?

Reply:
Nope, we didn't. The property is here to keep the type consistent